packages/transition-frontend/webpack.config.js

```diff
@@ -73,20 +83,28 @@ module.exports = (env) => {
           exclude: /node_modules/,
         },
         {
-          loader: 'json-loader',
+          use: 'json-loader',
           test: /\.geojson$/,
           include: includeDirectories
         },
         {
           test: /\.(ttf|woff2|woff|eot|svg)$/,
+/*
```

Comment:
Any reason to keep this commented code?

Reply:
No, this was leftover, I will remove it.